.github/workflows/_tests.yml

```
@@ -0,0 +1,36 @@
+name: Run tests
+on:
+  push:
+    branches: [ "*" ]
```

```suggestion
  push:
    branches: [ "master" ]
  pull_request:
    branches: [ "*" ]
```

I think we should use the same trigger as in `main.yaml` for uniformity.